src/Sulu/Bundle/AdminBundle/Resources/js/components/InfiniteScroller/tests/InfiniteScroller.test.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import {mount} from 'enzyme';
+import React from 'react';
+import InfiniteScroller from '../InfiniteScroller';
+
+window.getComputedStyle = jest.fn();
+
+test('InfiniteScroller traverses the dom upwards until it finds a scroll container', () => {
+    window.getComputedStyle.mockReturnValue({
+        'overflow-y': 'auto',
+    });
+
+    const loadSpy = jest.fn();
+    const infiniteScroller = mount(
+        <div id="scrollable">
+            <InfiniteScroller
+                onLoad={loadSpy}
+            >
+                <div />
+            </InfiniteScroller>
+        </div>
+    );
+
+    expect(infiniteScroller.find('InfiniteScroller').get(0).scrollContainer.id).toBe('scrollable');
+});
+
+test('InfiniteScroller should call onLoad if the the bottom of the content is reached', (done) => {
+    window.getComputedStyle.mockReturnValue({
+        'overflow-y': 'auto',
+    });
+
+    const loadSpy = jest.fn();
+    const infiniteScroller = mount(
+        <div id="scrollable">
+            <InfiniteScroller
+                total={10}
+                current={1}
+                onLoad={loadSpy}
+            >
+                <div />
+            </InfiniteScroller>
+        </div>
+    );
+
+    infiniteScroller.find('InfiniteScroller').get(0).scrollContainer = {
+        getBoundingClientRect: () => ({
+            bottom: 260,
+        }),
+    };
+    infiniteScroller.find('InfiniteScroller').get(0).elementRef = {
+        getBoundingClientRect: () => ({
+            bottom: 300,
+        }),
+    };
+    infiniteScroller.find('InfiniteScroller').get(0).unbindScrollListener = jest.fn();
+    infiniteScroller.find('InfiniteScroller').get(0).scrollListener();
```

Comment:
No, `scrollListener` is called when the container is resized or scrolled. I think calling the method is fine because I want to test my code and not if the DOM events work properly

Reply:
You shouldn't test if the DOM events work properly, but if the `InfiniteScroller` components binds correctly to the DOM.

Reply:
I am already mocking the `scrollContainer` some lines above that. I have to do that because `getBoundingClientRect` returns `0` if I use the `jsdom` implementation. I am not sure how to trigger `resize` or `scroll` here so that `scrollListener` will be called.